util/src/main/java/tc/oc/pgm/util/text/TextFormatter.java

```diff
@@ -120,6 +121,22 @@ public static Component horizontalLineHeading(
 
   public static Component horizontalLineHeading(
       CommandSender sender, Component text, TextColor lineColor) {
-    return horizontalLineHeading(sender, text, lineColor, ComponentUtils.MAX_CHAT_WIDTH);
+    return horizontalLineHeading(sender, text, lineColor, LegacyFormatUtils.MAX_CHAT_WIDTH);
+  }
+
+  /*
+   * Convert ChatColor -> TextColor
+   */
+  public static TextColor convert(Enum<?> color) {
+    return TextColor.valueOf(color.name());
+  }
+
+  /**
+   * Support kept for custom tab-list/nms usage (TODO: if tab-list system ever changes,
+   * upgrade/remove this)
+   */
+  public static BaseComponent renderBaseComponent(CommandSender viewer, Component component) {
+    return new net.md_5.bungee.api.chat.TextComponent(
+        TextTranslations.translateLegacy(component, viewer));
```

Comment:
Use `SpigotTextAdapter` to convert to a `BaseComponent`. This should fix the empty translations.